app/views/workflows/edit.html.erb

```diff
@@ -29,29 +29,31 @@ See doc/COPYRIGHT.rdoc for more details.
 <%= render :partial => 'action_menu' %>
 <h2><%= Workflow.model_name.human %></h2>
 <%= render :partial => 'layouts/action_menu_specific' %>
-<%= form_tag({}, :method => 'get') do %>
+<%= styled_form_tag({}, :method => 'get') do %>
   <fieldset class="simple-filters--container">
-    <legend>
-      <%=l(:text_workflow_edit)%>:
-    </legend>
+    <legend><%=l(:text_workflow_edit)%></legend>
     <ul class="simple-filters--filters">
-      <li class="simple-filters--filter">
-        <%= label_tag :role_id, Role.model_name.human %>
-        <%= select_tag 'role_id', options_from_collection_for_select(@roles, "id", "name", @role && @role.id) %>
-      </li>
-      <li class="simple-filters--filter">
-        <%= label_tag :type_id, Type.model_name.human %>
-        <%= select_tag 'type_id', options_from_collection_for_select(@types, "id", "name", @type && @type.id) %>
+      <li class="simple-filters-filter">
+        <%= styled_label_tag :role_id, Role.model_name.human, class: 'simple-filters--filter-name' %>
+        <div class="simple-filters--filter-value">
+          <%= styled_select_tag 'role_id', options_from_collection_for_select(@roles, "id", "name", @role && @role.id) %>
+        </div>
+        <div class="form--field">
+          <%= hidden_field_tag 'used_statuses_only', '0', id: 'used_statuses_only_hidden' %>
+
+          <%= styled_label_tag :used_statuses_only, class: 'form--label-with-check-box' do %>
```

Comment:
using the `styled_label_tag` helper here unfortunately adds the `form--label` class as well which does not play nicely with the `form--label-with-check-box` class as it limits the length of the label.

![image](https://cloud.githubusercontent.com/assets/617519/6864603/3eef5ec2-d464-11e4-97fe-7a83e52eb559.png)


Options I see:

1) use `label_tag` instead
2) change the DOM slightly to use the `styled_label_tag` with the `-trailing-label` modifier class applied.